store/store.go

```diff
@@ -298,6 +298,10 @@ func (s *store) CompareAndSwap(nodePath string, prevValue string, prevIndex uint
 		return nil, err
 	}
 
+	if expireOpts.Refresh {
```

Comment:
should this return an error if value != "" in case someone expects this to use the given `value` like a refreshless CAS?